packages/react-router/tests/link.test.tsx

```diff
@@ -1095,6 +1095,33 @@ describe('Link', () => {
     expect(errorText).toBeInTheDocument()
   })
 
+  test('when navigating to the root with an error in component', async () => {
+    const notFoundComponent = vi.fn()
+
+    const rootRoute = createRootRoute({
+      errorComponent: () => <span>Oops! Something went wrong!</span>,
```

Comment:
Please use different error messages both in the `errorComponent` and the `component` so that anyone reading the test knows exactly what the expected outcome is.

Something like:
- `errorComponent` - "Rendering errorComp message"
- `component` - "Throwing from route component"

Feel free to change the exact verbiage as you please. Just please make them very distinctly unique.